### DISCUSSION TITLE
How to use Vite module bundler and jQuery?

### DISCUSSION THREAD

**Discussion by indramal** [jquery/jquery#5268] [Q&A]
How to use jQuery module after using npm installation and get optimum JS file with build. I can see there is jQuery modules but didn't know how to work with it.

<sub>Edited by @mgol to remove typos</sub>

**Comment by mgol** [comment]
Can you explain what exact problems do you have? Most bundlers should work with a regular:
```js
import $ from 'jquery';
```

If you're looking for more modular builds then that's harder to achieve in jQuery due to its architecture; in jQuery 4 it will be possible to import individual jQuery modules, though, via e.g.:
```js
import $ from 'jquery/src/event.js';
```

**Reply by indramal** [reply]
As you said, JQuary can not use with module bundler. I think individual modules is better option because to reduce the size.
